packages/router-plugin/tests/code-splitter/test-files/function-as-parameter.tsx

```diff
@@ -8,22 +8,22 @@ const ReactUseMemoCall1 = React.useMemo(function performAction() {
   return 'true'
 }, [])
 
-console.log(ReactUseMemoCall1)
+console.info(ReactUseMemoCall1)
```

Comment:
Why are these tests being changed? No real big deal, just wondering?

Reply:
I changed to info so I could more easily find lingering console logs from development across the code base.